mungegithub/mungers/blunderbuss.go

```diff
@@ -62,6 +64,11 @@ func (b *BlunderbussMunger) RequiredFeatures() []string {
 
 // Initialize will initialize the munger
 func (b *BlunderbussMunger) Initialize(config *github.Config, features *features.Features) error {
+	whitelist, err := config.ValidAssignees()
```

Comment:
Doesn't this mean that it will get set when the mungegithub launches and never again? if it runs for days/weeks it would never get an update?

Should we make this something like a 'feature' and have it update every loop?